lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -124,10 +124,11 @@
                             view: new AccountSettingsFieldViews.DropdownFieldView({
                                 model: userAccountModel,
                                 required: true,
-                                title: gettext('Country or Region'),
+                                title: gettext('Country of Full-Time Residence'),
                                 valueAttribute: 'country',
                                 options: fieldsData.country.options,
-                                persistChanges: true
+                                persistChanges: true,
+                                helpMessage: gettext('This field represents the country of full-time residence not the country of origin') // eslint-disable-line max-len
```

Comment:
It was mentioned on the ticket that this can benfit from a help text so i added a help text. you can get a review from some one appropriate to make it batter.